src/ApplicationRunner.php

```diff
@@ -151,7 +109,7 @@ protected function getConfig(): ConfigInterface
      */
     protected function getContainer(): ContainerInterface
     {
-        $this->container ??= $this->createDefaultContainer($this->getConfig(), $this->containerConfigGroup);
+        $this->container ??= $this->createDefaultContainer();
```

Comment:
Shound't we stil pass params?

Reply:
No, see https://github.com/yiisoft/yii-runner/pull/39#discussion_r1083255170